core/src/main/java/org/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -415,6 +415,14 @@
    */
   public static final String AGGREAGATE_COLUMNAR_KEY_BLOCK_DEFAULTVALUE = "true";
   /**
+   * TIME_STAT_UTIL_TYPE
+   */
+  public static final String TIME_STAT_OPEN = "time.statistics.open";
```

Comment:
Name looks not understandable. Better change as `enable.data.loading.statistics`